.github/workflows/dotnet.yml

```
@@ -13,7 +13,7 @@ jobs:
       - uses: actions/checkout@v4
       - uses: actions/setup-node@v4
         with:
-          node-version: 16
+          node-version: 18
```

what if you made all of these point to the top-level `package.json`?

```suggestion
          node-version-file: package.json?
```